boilerplate/app/navigators/app-navigator.tsx

```diff
@@ -53,7 +53,11 @@ interface NavigationProps extends Partial<React.ComponentProps<typeof Navigation
 
 export const AppNavigator = (props: NavigationProps) => {
   const colorScheme = useColorScheme()
-  useBackButtonHandler(canExit)
+
+  // What route names do we allow the back button to exit the app from?
+  const exitRoutes = ["welcome"]
+  useBackButtonHandler((routeName) => exitRoutes.includes(routeName))
```

Comment:
Simplifying this file a bit, since the `canExit` function is no longer used anywhere else.